spec/formatting.md

```diff
@@ -677,8 +677,8 @@ These are divided into the following categories:
     > ```
 
   - A **_Missing Selector Annotation error_** is an error that occurs when the _message_
-    contains a _selector_ that does not have an _annotation_,
-    or contains a _variable_ that does not directly or indirectly reference a _declaration_ with an _annotation_.
+    contains a _selector_ that does not directly or indirectly reference
+    an _expression_ with an _annotation_.
 
     > Example invalid messages resulting in a _Missing Selector Annotation error_:
```

Comment:
Making the wording flow a bit better

```suggestion
> **Example**.
> Here are some invalid _messages_ that would result in a _Missing Selector Annotation error_:

Reply:
This text is modeled on the existing text for the other kinds of data model errors. Perhaps a separate PR could add this change in a way that's consistent for all the error descriptions?